### PR TITLE
Move NuGetRepack from RoslynTools to Arcade

### PR DESCRIPTION
**Pull request by JoeRobich**

### CODE REVIEW

**Comment by jonfortescue** [comment]
@JoeRobich what's the status on this?

File: eng/Versions.props [hunk 1]
```diff
@@ -37,8 +37,8 @@
     <McMasterExtensionsCommandLineUtils>2.3.0</McMasterExtensionsCommandLineUtils>
     <NewtonsoftJsonVersion>9.0.1</NewtonsoftJsonVersion>
     <MicrosoftBclJsonSourcesVersion>4.6.0-preview4.19202.2</MicrosoftBclJsonSourcesVersion>
-    <NuGetVersioningVersion>4.4.0</NuGetVersioningVersion>
     <NuGetVersion>4.9.0-rtm.5658</NuGetVersion>
+    <NuGetVersioningVersion>$(NuGetVersion)</NuGetVersioningVersion>
```
Comment (tmat) [inline eng/Versions.props]: ><NuGetVersioningVersion>$(NuGetVersion)</NuGetVersioningVersion [](start = 4, length = 63)

Why do we need to change this version? Seems like the version in roslyn-tools was also 4.4.0

File: src/Microsoft.DotNet.Arcade.Sdk/tools/BuildReleasePackages.targets [hunk 2]
```diff
@@ -1,11 +1,5 @@
 <!-- Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE file in the project root for more information. -->
 <Project>
-  <PropertyGroup>
-    <_NuGetRepackAssembly Condition="'$(MSBuildRuntimeType)' != 'Core'">$(NuGetPackageRoot)roslyntools.nugetrepack.buildtasks\$(RoslynToolsNuGetRepackVersion)\tools\net472\RoslynTools.NuGetRepack.BuildTasks.dll</_NuGetRepackAssembly>
-    <_NuGetRepackAssembly Condition="'$(MSBuildRuntimeType)' == 'Core'">$(NuGetPackageRoot)roslyntools.nugetrepack.buildtasks\$(RoslynToolsNuGetRepackVersion)\tools\netcoreapp2.1\RoslynTools.NuGetRepack.BuildTasks.dll</_NuGetRepackAssembly>
-  </PropertyGroup>
```
Comment (tmat) [inline src/Microsoft.DotNet.Arcade.Sdk/tools/BuildReleasePackages.targets]: Include

File: src/Microsoft.DotNet.Arcade.Sdk/tools/DefaultVersions.props [hunk 3]
```diff
@@ -43,7 +43,6 @@
     <UsingToolIbcOptimization Condition="'$(UsingToolIbcOptimization)' == ''">false</UsingToolIbcOptimization>
     <UsingToolVisualStudioIbcTraining Condition="'$(UsingToolVisualStudioIbcTraining)' == ''">false</UsingToolVisualStudioIbcTraining>
     <UsingToolNetFrameworkReferenceAssemblies Condition="'$(UsingToolNetFrameworkReferenceAssemblies)' == ''">false</UsingToolNetFrameworkReferenceAssemblies>
-    <UsingToolNuGetRepack Condition="'$(UsingToolNuGetRepack)' == ''">false</UsingToolNuGetRepack>
```
Comment (tmat) [inline src/Microsoft.DotNet.Arcade.Sdk/tools/DefaultVersions.props]: > <UsingToolNuGetRepack Condition="'$(UsingTool [](start = 3, length = 46)

Also needed

File: src/Microsoft.DotNet.Arcade.Sdk/tools/BuildReleasePackages.targets (hunk 2 above)
Comment (tmat) [reply src/Microsoft.DotNet.Arcade.Sdk/tools/BuildReleasePackages.targets]: `$(ArcadeSdkVersion)`

---
In reply to: [308487549](https://github.com/dotnet/arcade/pull/3120#discussion_r308487549) [](ancestors = 308487549)

**Comment by JoeRobich** [comment]
@tmat I've run this locally against the Roslyn repo and it built Release and Prerelease packages as expected. It is ready for another review. Thanks!

**Comment by tmat** [comment]
Thanks!

**Pull request merged**